modules/running-compliance-scans-worker-node.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-scans.adoc
+
+[id="running-compliance-scans-worker-node_{context}"]
+=  Scheduling the result server pod on a worker node
+
+The result server pod is used to mount the persistent volume (PV) storage volume that stores the raw Asset Reporting Format (ARF) scan results. The `nodeSelector` and `tolerations` attributes allow you to configure the location of the result server pod.
+
+This is helpful for those environments where control plane nodes are not permitted to mount persistent volumes.
+
+.Procedure
+. Create a `ScanSetting` custom resource (CR) for the Compliance Operator:
```

Comment:
Should there be a command at the end of this sentence? If not, then replace : with a period.